content/en/docs/Examples/p2p_e2e.md

```diff
@@ -141,9 +138,17 @@ Afterward, you should be able to ssh to one of the machines and be able to use y
 ``` bash
 $ ssh kairos@IP
 $ sudo su -
+$ k9s
```

Comment:
all flavors should now automagically pick up the kairos config, in case it doesn't I added the alert but trying to keep the steps as simple as possible